data/i18n/ja/ja.toml

```diff
@@ -1,6 +1,12 @@
 # i18n strings for the English (main) site.
 # NOTE: Please keep the entries in alphabetical order when editing
 
+[docs_version_latest_heading]
```

Comment:
The order this is presented in the other toml files for other languages is eg.:

```
data/i18n/en/en.toml:116:[docs_version_current]
data/i18n/en/en.toml-117-other = "(this documentation)"
data/i18n/en/en.toml-118-
data/i18n/en/en.toml:119:[docs_version_latest_heading]
data/i18n/en/en.toml-120-other = "Latest version"
data/i18n/en/en.toml-121-
data/i18n/en/en.toml:122:[docs_version_other_heading]
data/i18n/en/en.toml-123-other = "Older versions"
data/i18n/en/en.toml-124-
```
We might want to keep the same order

Reply:
Done, thank you.